lib/thor/shell/basic.rb

```diff
@@ -143,14 +143,14 @@ def say_status(status, message, log_status = true)
         stdout.flush
       end
 
-      # Make a question the to user and returns true if the user replies "y" or
+      # Make a question to the user and returns true if the user replies "y" or
```

Comment:
While we are changing this we can improve the sentences a bot as well:
```suggestion
      # Asks the user a question and returns true if the user replies "y" or
```

Reply:
Thank you for your review! I fixed it.